pyproject.toml

```diff
@@ -22,7 +22,7 @@ packages = [
 python = "^3.7"
 nautobot = "^1.2.0"
 nautobot-ssot = "^1.0.1"
-cloudvision = "^1.5.0"
+cloudvision = "1.9.0"
```

Comment:
We may want to [caret](https://python-poetry.org/docs/dependency-specification/#caret-requirements) this to support anything up to 2.0. Thoughts @jdrew82?

Reply:
Agreed, assuming they're following semantic versioning.